plugins/ufc-scoreboard/manager.py

```
@@ -0,0 +1,1206 @@
+        if not self.is_enabled:
+            return
+
+        try:
+            if self.ufc_enabled:
+                if hasattr(self, "ufc_live"):
+                    self.ufc_live.update()
+                if hasattr(self, "ufc_recent"):
+                    self.ufc_recent.update()
+                if hasattr(self, "ufc_upcoming"):
+                    self.ufc_upcoming.update()
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)